swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -183,6 +183,7 @@ paths:
         - $ref: "#/components/parameters/consentId_HEADER_optional"
         - $ref: "#/components/parameters/PSU-IP-Address_mandatory"
         - $ref: "#/components/parameters/TPP-Redirect-Preferred"
+        - $ref: "#/components/parameters/TPP-Decoupled-Preferred"
```

Comment:
Relevant Change: TPP-Decoupled-Preferred is new
Adoption approach: Follow the change